geth/account/adapter_test.go

```diff
@@ -0,0 +1,37 @@
+package account
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/whisper/whisperv5"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestProxy_Base_Success(t *testing.T) {
```

Comment:
Here maybe is an example of my biggest troubles with this PR. 

- We create empty type instances
- What does the mock calls do then? I have to admit I've never been a fan of generated mocks. To avoid integration with lower levels I always preferred stubs (see https://martinfowler.com/articles/mocksArentStubs.html) injected manually and returning predefined answers so that our unit tests can focus on the tested unit.
- What is tested by the direct access to `m.node`?

Here is one reason why I also favour commenting in tests. Function names like here are often not enough for maintenance by others later. And inside the tests comments like

```
// Preparing by ...

...

// Testing that ...
```

May seem bloated, but does help others.